components/form/FCreditCardInput.vue

```diff
@@ -248,15 +276,16 @@ watch(value, newValue => {
 
   if (
     (newCreditCard && newCreditCard.lengths.includes(spacelessValue.length)) ||
-    (!newCreditCard && spacelessValue.length === 16)
+    (!newCreditCard &&
+      spacelessValue.length === defaultMask.split('#').length - 1)
   ) {
     // Manually trigger validation if :
-    // - matched credit card length meets the input value
-    // -  or if no credit card was found and input value length is 16
+    // - Input value length matches one of the lengths of the matched credit card
+    // - No credit card was found but input value length reached 16 digits
```

Comment:
Update ton commentaire 
```suggestion
    // - No credit card was found but input value length reached the number of digits of default mask
```